readthedocs/docsitalia/urls.py

```diff
@@ -20,6 +24,7 @@
 
 
 urlpatterns = [
+    url(r'^docsitalia/', include(docsitalia_urls)),
```

Comment:
I needed to move this up because otherwise a rtd test failed:
`________________________ APIUnauthAccessTest.test_api_urls _________________________

self = <readthedocs.rtd_tests.tests.test_privacy_urls.APIUnauthAccessTest testMethod=test_api_urls>
get_public_task_data = <MagicMock name='get_public_task_data' id='139883792972240'>

    @mock.patch('readthedocs.restapi.views.task_views.get_public_task_data')
    def test_api_urls(self, get_public_task_data):
        from readthedocs.restapi.urls import urlpatterns
        get_public_task_data.side_effect = TaskNoPermission('Nope')
>       self._test_url(urlpatterns)

rtd_tests/tests/test_privacy_urls.py:374: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
rtd_tests/tests/test_privacy_urls.py:127: in _test_url
    self.assertResponse(path=path, name=name)
rtd_tests/tests/test_privacy_urls.py:83: in assertResponse
    value=resp_val))
E   AssertionError: Attribute mismatch for view api-root (/docsitalia/): status_code != 200 (got 404)
`
This was due the fact that the api root didn't worked

Reply:
This does not make any sense to me :)

Reply:
Yes because the api root was treated as a non exisiting publisher if I remember correctly